syntax = "proto3";
package echo;
//option py_generic_services = true;
option cc_generic_services = true;
option java_generic_services = true;
option java_package = "echo";
option java_outer_classname = "EchoProto";

message EchoRequest {
  int32 id=1;
  int32 cnt=2;
}

message EchoResponse {
  string payload = 2;
}

service EchoService {
  rpc Echo (EchoRequest) returns (EchoResponse);
}

